rootfs/bin/test-unit

```diff
@@ -8,5 +8,5 @@ set -eou pipefail
 
 sudo -u postgres "$PGBIN"/pg_ctl -D "$PGDATA" -l /tmp/logfile start
 python3 manage.py check
-coverage run manage.py test --settings=api.settings.testing --noinput registry api scheduler.tests
```

Comment:
Looks like parallel speeds up things but is not recommended by the Django maintainers because running in a single process is more accurate. Can we remove if for now?

> Coverage should be run in a single process to obtain accurate statistics. To run coverage on the Django test suite using the standard test settings:

```
$ coverage run ./runtests.py --settings=test_sqlite --parallel=1
```

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/unit-tests/#code-coverage

Reply:
Thanks!